sdk/tests/deqp/framework/opengl/simplereference/sglrReferenceContext.js

```diff
@@ -171,7 +175,7 @@ goog.scope(function() {
 
         // \todo [2013-08-01 pyry] Do we want to make these conditional based on renderCtx?
         addExtension("gl.EXT_color_buffer_half_float");
-        addExtension("gl.EXT_color_buffer_float");
+        addExtension("gl.WEBGL_color_buffer_float");
```

Comment:
The planned direction is to expose EXT_color_buffer_float on WebGL 2.0 contexts, and to not move forward with the WebGL 1.0 extensions for this functionality (WEBGL_color_buffer_float, EXT_color_buffer_half_float).